vars/setupCommonPipelineEnvironment.groovy

```diff
@@ -11,7 +11,10 @@ import groovy.transform.Field
 
 @Field Set GENERAL_CONFIG_KEYS = [
     /** */
-    'collectTelemetryData'
+    'collectTelemetryData',
+
+    /** Credentials (username and password) used to download custom defaults if access is secured.*/
+    'customDefaultCredentialsId'
```

Comment:
Hm. I think it may even need two plural 's'. The feature's name is "customDefaults" and those are the "credentials" for it. So `customDefaultsCredentialsId`. What do you think?